lesson_02/ht_template/job2/dal/read_sales.py

```diff
@@ -0,0 +1,17 @@
+from typing import List, Dict, Any
+import json
+import os
+
+
+def get_sales_from_file(raw_dir: str, date: str) -> List[Dict[str, Any]]:
```

Comment:
Nothing related to actual sales data is used here. You can use this function for any JSON data.
I would name this file `local.py` and rename this function to `read_json_data` or something similar